po/de.po

```diff
@@ -2384,6 +2391,10 @@ msgid ""
 "reports for expenses and income, an income-expense chart, and an account "
 "summary."
 msgstr ""
+"Ein neues Berichts-Untermenü <b>Mehrspaltig</b> enthält den alten "
+"benutzerdefinierten mehrspaltigen Bericht und einen neuen <b>Dashboard</b>-"
```

Comment:
Dashboard sollte ins Glossar und wir sollten was besseres suchen. Google translate sagt "Instrumententafel".

Reply:
@fellen: Ich zitiere mal aus einer Diskussion auf leo.org: "Der Begriff "Dashboard" (auch so auf Deutsch) ist weit verbreitet für Übersichtsseiten, die oftmals verschiedene Berichte, Kennzahlen und Anzeigen konsolidieren."

Ich denke, Dashboard ist in der deutschen Sprache schon angekommen; auf duden.de gibt es sogar den Genitiv und den Plural.

Und ansonsten können wir im Glossar die zitierte Beschreibung verwenden.

Reply:
@fellen: habe ich im Glosser mal wie oben zitiert eingefügt.

Reply:
>  "Der Begriff "Dashboard" (auch so auf Deutsch) ist weit verbreitet für Übersichtsseiten, die oftmals verschiedene Berichte, Kennzahlen und Anzeigen konsolidieren."
> 
> Ich denke, Dashboard ist in der deutschen Sprache schon angekommen; auf duden.de gibt es sogar den Genitiv und den Plural.

Vielleicht bei den Marketing-Leuten, die einem ständig alten, grischisch-römischen Wein in neuen anglizistischen Schläuchen zu verkaufen versuchen. Ich muß immer Google-Translate anschmeißen. ;-)

Reply:
Im Zweifelsfall Meinungsbild auf [gnucash-de](mailto:gnucash-de) erstellen. :-)

Reply:
> @fellen: habe ich im Glosser mal wie oben zitiert eingefügt.

Das war das falsche Glossar. Wir wollen erstmal unser Glossary halbwegs komplett haben. Danach können die Teams, die es übersetzt haben, es ins Weblate-Glossary hochladen. Schick es mir doch als PR für https://github.com/Gnucash/gnucash/blob/maint/po/glossary/gnc-glossary.txt.
Infos zur Datei: https://wiki.gnucash.org/wiki/Translation#Terms_missing_or_inadequate_in_the_glossary_file